include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge_sort.h

```diff
@@ -57,6 +56,152 @@ struct __leaf_sort_kernel
     }
 };
 
+struct __group_merge_path_sorter
+{
+    template <typename _StorageAcc, typename _Compare>
+    bool
+    sort(const sycl::nd_item<1>& __item, const _StorageAcc& __storage_acc, _Compare __comp, std::uint32_t __start,
```

Comment:
Let's declare not changed params as const?

Reply:
Done

Reply:
We discussed it offline and came to an agreement that it was optional.